.github/workflows/linux-wpt.yml

```diff
@@ -62,75 +64,64 @@ jobs:
         run: |
           ./mach update-wpt --sync --patch
       - name: Run tests
-        if: ${{ !inputs.wpt-sync-from-upstream }}
         run: |
-          python3 ./mach test-wpt $WPT_COMMAND_LINE_ARG ${{ inputs.wpt-tests-to-run }} \
+          mkdir -p wpt-filtered-logs/linux-${{ inputs.wpt-layout }}
+          mkdir -p wpt-full-logs/linux-${{ inputs.wpt-layout }}
+          python3 ./mach test-wpt \
+            $WPT_LAYOUT_ARG \
+            $WPT_ALWAYS_SUCCEED_ARG \
+            ${{ inputs.wpt-tests-to-run }} \
             --${{ inputs.profile }} --processes $(nproc) --timeout-multiplier 2 \
             --total-chunks ${{ env.max_chunk_id }} --this-chunk ${{ matrix.chunk_id }} \
-            --log-raw test-wpt.${{ matrix.chunk_id }}.log \
-            --log-raw-unexpected unexpected-test-wpt.${{ matrix.chunk_id }}.log \
-            --filter-intermittents filtered-test-wpt.${{ matrix.chunk_id }}.json
+            --log-raw wpt-full-logs/linux-${{ inputs.wpt-layout }}/${{ matrix.chunk_id }}.log \
+            --log-raw-unexpected wpt-filtered-logs/linux-${{ inputs.wpt-layout }}/${{ matrix.chunk_id }}.log \
+            --filter-intermittents wpt-filtered-logs/linux-${{ inputs.wpt-layout }}/${{ matrix.chunk_id }}.json
```

Comment:
How does sync handle those?

Reply:
Sync uses the full logs which are produced unconditionally.

Reply:
There is a lot of [`Conflicting metadata values for ...`](https://github.com/mrobinson/servo/actions/runs/7941029162/job/21683631506#step:8:12) that didn't happened before because some tests are run/logged multiple times for filtering. While I do not think this is blocker, solving this problem would be nice followup (maybe using filtered results for wpt-update would reduce flakes in WPT import PRs).

Reply:
Hrm. This happens when a test flakes (it's run twice, but has different results each time) and is the expected behavior during try runs. Now that filtering intermittents is turned on unconditionally, tests that fail are run twice and have their results appended to the log each time. I think this will make the generated results more reliable -- because tests that flake won't have their results updated by the flaky test run during the WPT import.

On the other hand, if a new test flakes (much less likely, but possible, no results will be generated for it). I guess in those cases, we will need to manually generate results to have the WPT import land. We have to do this anyway and since the amount of flaky tests is bigger than the intersection of flaky tests and new tests, this should hopefully make imports easier.

Reply:
If this looks good, to you @sagudev, maybe we can land it today and see how the WPT import goes. If there is trouble, we can either back it out or fix the problem once we see how it goes. What do you think?

Reply:
> I think this will make the generated results more reliable -- because tests that flake won't have their results updated by the flaky test run during the WPT import

Will it really though? Because we currently feed full not filtered logs into wpt-update or am I missing something?

Reply:
The way filtering mode works is that the full logs have data from all runs, but json files are produced in parallel that contain information about what tests are filtered by the intermittent dashboard. This json file doesn't affect the full log though.